app/src/main/res/layout/item_thread.xml

```diff
@@ -15,128 +15,134 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout contains a lot of layouts, can't we simplify everything?

Reply:
Will be handled by this issue: https://github.com/Infomaniak/android-mail/issues/142